extensions/interactions/MultipleChoiceInput/MultipleChoiceInput.py

```diff
@@ -42,7 +42,7 @@ class MultipleChoiceInput(base.BaseInteraction):
     # Radio buttons get unselected when specifying a solution. This needs to be
     # fixed before solution feature can support this interaction.
     can_have_solution: bool = False
-    show_generic_submit_button: bool = False
+    show_generic_submit_button: bool = True
```

Comment:
Is a comment required above this to make sure no one changes it back to 'False' ?

Reply:
Done